test/e2e/pages/positronAssistant.ts

```diff
@@ -22,7 +22,7 @@ const GEMINI_BUTTON = 'button.positron-button.language-model.button:has(#google-
 const COPILOT_BUTTON = 'button.positron-button.language-model.button:has(#copilot-provider-button)';
 const CHAT_PANEL = '#workbench\\.panel\\.chat';
 const RUN_BUTTON = 'a.action-label.codicon.codicon-play[role="button"][aria-label="Run in Console"]';
-// const OATH_RADIO = '.language-model-authentication-method-container input#oauth[type="radio"]';
+const OATH_RADIO = '.language-model-authentication-method-container input#oauth[type="radio"]';
```

Comment:
nit for searchability of `oauth`

```suggestion
const OAUTH_RADIO = '.language-model-authentication-method-container input#oauth[type="radio"]';
```